src/main/java/net/glowstone/generator/structures/GlowDungeon.java

```diff
@@ -96,9 +96,9 @@ public boolean generate(World world, Random random, StructureBoundingBox genBoun
 
         for (int x = 0; x < sizeX; x++) {
             for (int z = 0; z < sizeZ; z++) {
-                for (int y = HEIGHT - 2; y >= 0; y--) {
+                for (int y = HEIGHT - 1; y >= 0; y--) {
                     BlockState state = builder.getBlockState(new Vector(x, y, z));
-                    if (y > 0 && x > 0 && z > 0 && x < sizeX - 1 && z < sizeZ - 1) {
+                    if (y > 0 && x > 0 && z > 0 && x < sizeX - 1 && y < HEIGHT - 2 && z < sizeZ - 1) {
```

Comment:
since this is putting air in the room, shouldnt the bounds for y be **1** to HEIGHT - 2? (and since the for loop starts at < HEIGHT - 2 already, i think we can remove that check)